assets/css/layout.less

```diff
@@ -21,7 +21,7 @@ header {
   #open_main_menu {
     color: @button-main-menu;
     cursor: pointer;
-    padding: 8px 4px;
+    padding: 8px;
```

Comment:
This commit name is not related to the action taken. 
It is fine for now, but in a future, make sure that the commit message applied (in this case "removed important property") summarises what was done.